packages/flutter/lib/src/material/stepper.dart

```diff
@@ -232,6 +240,27 @@ class Stepper extends StatefulWidget {
   /// The index into [steps] of the current step whose content is displayed.
   final int currentStep;
 
+  /// give background color to header
+  final Color? backgroundHeaderColor;
+
+  /// set the BoderRadius to header
+  final BorderRadiusGeometry? headerRadius;
+
+  /// set the color to text of active step
+  final Color? activeCircleTextColor;
+
+  /// set the background color to step
+  final Color? circleColor;
+
+  /// set the underline color
+  final Color? lineColor;
```

Comment:
This valuable currently only affects the [StepperType.horizontal] mode, but the vertical arrangement does also have a line. I would expect a variable like this to affect both modes, since the same artifact appears in both.